_source/_posts/2021-09-16-spring-native-okta-starter.adoc

```diff
@@ -15,6 +15,8 @@ tweets:
 - "Use Spring Native to create an app with @springboot + @okta. It's pretty slick!"
 image: blog/spring-native-okta-starter/spring-native-okta.png
 type: conversion
+changelog:
+- 2021-01-21: Updated to use `.okta.env` instead of `application.properties`. This change is necessary since the okta-spring-boot-sample now uses [dotenv to load Okta settings](https://github.com/okta-samples/okta-spring-boot-sample/pull/22). Changes to this post can be viewed in [okta-blog#1049](https://github.com/oktadev/okta-blog/pull/1049).
```

Comment:
```suggestion
- 2021-01-25: Updated to use `.okta.env` instead of `application.properties`. This change is necessary since the okta-spring-boot-sample now uses [dotenv to load Okta settings](https://github.com/okta-samples/okta-spring-boot-sample/pull/22). Changes to this post can be viewed in [okta-blog#1049](https://github.com/oktadev/okta-blog/pull/1049).
```